### ISSUE TITLE
Remember app theme from account

### ISSUE THREAD

**Issue by test2a** [stom79/TubeLab#118]
1.9
Theme change still has crashes making you login again. Anyways, the default app theme is white and thats okay.
What I have in mind is, we can set a theme for an instance in the account.
There is are default 
* peertube default
* peertube dark
* instance default

If the api is there, can we check if dark is selected and if so, set the app theme to dark ? Its probably too much to ask but yeah.

Anyways, the update is so much better now. Subscriptions for example now successfully open but channel profile still shows no videos. Its still much better

**Comment by stom79** [comment]
There are three issues here :smiley:

**Comment by test2a** [comment]
Nah. I was just explaining. Check if theme is set to dark on web and accordingly do that in app
